### PR TITLE
787XX delegate release notes

### PR DESCRIPTION
**Pull request by KimberlyFields**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — DISMISSED** [review]
LGTM

File: release-notes/delegate.md [hunk 1]
```diff
@@ -14,6 +14,33 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+This release does not include any early access features.
+
+### Fixed issues
+
+A pipeline stalled with only one ServiceNow task running. (DEL-6042)
+
+This issue was fixed with the following updates:
+
+- Explicitly fail tasks that were never assigned to a delegate after 4 successful broadcast attempts per delegate to all eligible delegates in the account.
```
Comment (StrawberryPad) [inline release-notes/delegate.md:41]: I read this sentence a dozen times to try to understand it. Not sure if this is what it's trying to say:
"When a task is not assigned to a delegate, the (system?) explicitly fails the task after running 4 successful broadcast attempts to each eligible delegate in the account."

Or something like it that makes sense in the context.

File: release-notes/delegate.md [hunk 2]
```diff
@@ -14,6 +14,33 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+### What's new
+
+- Integrated **Logs** API in the **Executed Details** page where the delegate task ID is available. (DEL-6035)
+
+  You can now view logs for delegate tasks for pipeline steps that are running or finished. This can help with debugging issues. 
+
+- Set an expiry for delegate tokens. (DEL-5652)
+
+  When you create a delegate token through APIs, you can provide an optional parameter `revokeAfter`, which is the epoch time in milliseconds after which the token is marked as revoked. There can be a delay of up to one hour from when the epoch value is provided to when the token is revoked. 
+
```
Comment (StrawberryPad) [inline release-notes/delegate.md:30]: Just to break up the mouthful sentence:
When you create a delegate token through APIs, you can provide an optional parameter `revokeAfter`. This is the epoch time in milliseconds after which the token is marked as revoked. There can be a delay of up to one hour from when the epoch value is provided to when the token is revoked.

File: release-notes/whats-new.md [hunk 3]
```diff
@@ -16,13 +16,25 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+### Harness Delegate
+
+- Integrated **Logs** API in the **Executed Details** page where the delegate task ID is available. (DEL-6035)
+
+  You can now view logs for delegate tasks for pipeline steps that are running or finished. This can help with debugging issues. 
+
+- Set an expiry for delegate tokens. (DEL-5652)
+
+  When you create a delegate token through APIs, you can provide an optional parameter `revokeAfter`, which is the epoch time in milliseconds after which the token is marked as revoked. There can be a delay of up to one hour from when the epoch value is provided to when the token is revoked. 
+
```
Comment (StrawberryPad) [inline release-notes/whats-new.md:30]: same suggestion:
When you create a delegate token through APIs, you can provide an optional parameter `revokeAfter`. This is the epoch time in milliseconds after which the token is marked as revoked. There can be a delay of up to one hour from when the epoch value is provided to when the token is revoked.

**Review by StrawberryPad — APPROVED** [review]
Approved with comments

File: release-notes/delegate.md (hunk 2 above)
Comment (KimberlyFields) [reply release-notes/delegate.md:30]: Done

File: release-notes/delegate.md (hunk 1 above)
Comment (KimberlyFields) [reply release-notes/delegate.md]: Revised to: Tasks that were never assigned to a delegate explicitly fail after 4 successful broadcast attempts per delegate, to all eligible delegates in the account.

File: release-notes/whats-new.md (hunk 3 above)
Comment (KimberlyFields) [reply release-notes/whats-new.md:30]: Done

File: release-notes/delegate.md [hunk 4]
```diff
@@ -14,6 +14,33 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes). For Self-Managed Enterprise Edition, see [Self-Managed Enterprise Edition release notes](/release-notes/self-managed-enterprise-edition).
 
+## March 15, 2023, Harness version 78712, Harness Delegate version 78700
+
+Harness NextGen release 78712 includes the following changes for the Harness Delegate.
```
Comment (roshnisarangadharan) [inline release-notes/delegate.md:19]:
```suggestion
Harness NextGen release 78712 includes the following changes for the Harness delegate.
```

File: release-notes/delegate.md [hunk 5]
```diff
@@ -14,6 +14,33 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+This release does not include any early access features.
+
+### Fixed issues
+
+A pipeline stalled with only one ServiceNow task running. (DEL-6042)
+
+This issue was fixed with the following updates:
+
+- Tasks that were never assigned to a delegate explicitly fail after 4 successful broadcast attempts per delegate, to all eligible delegates in the account.
```
Comment (roshnisarangadharan) [inline release-notes/delegate.md:41]: "to all eligible delegates in the account" is not clear.

File: release-notes/whats-new.md [hunk 6]
```diff
@@ -16,13 +16,25 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 15, 2023 version 78712
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:19]: March 15, 2023, 
should there be a harness and delegate version like the other files?

File: release-notes/whats-new.md [hunk 7]
```diff
@@ -16,13 +16,25 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  When you create a delegate token through APIs, you can provide an optional parameter `revokeAfter`. This is the epoch time in milliseconds after which the token is marked as revoked. There can be a delay of up to one hour from when the epoch value is provided to when the token is revoked. 
+
 ## March 9, 2023
 
 ### Security Testing Orchestration
 
 * The new [Jira integration](/docs/security-testing-orchestration/use-sto/jira-integrations) has been enhanced. If an issue has an associated Jira ticket, the Issue Details pane now shows the ticket status along with the number. (STO-5491)
 
-## March 8, 2023. version 78619
+## March 8, 2023 version 78619
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:37]: March 8, 2023, version 78619

**Review by roshnisarangadharan — DISMISSED** [review]
Approved with comments.

**Review by deepakbolangady — APPROVED** [review]
Approving after resolving a merge conflict.

File: release-notes/delegate.md (hunk 4 above)
Comment (KimberlyFields) [reply release-notes/delegate.md:19]: Delegate is capitalized when Harness is in front of it.

File: release-notes/whats-new.md (hunk 6 above)
Comment (KimberlyFields) [reply release-notes/whats-new.md:19]: Not in this case.

File: release-notes/whats-new.md [hunk 8]
```diff
@@ -16,13 +16,25 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 15, 2023 version 78712
+
+### Harness Delegate
+
+- Integrated **Logs** API in the **Executed Details** page where the delegate task ID is available. (DEL-6035)
```
[the remark this answers is not in the document]
Comment (KimberlyFields) [reply release-notes/whats-new.md:23]: The suggested revision is not an accurate description of the change.

**Pull request merged**